src/PushNotifications.Api/Controllers/Subscriptions/Commands/PushyUnSubscriptionController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Elders.Cronus.DomainModeling;
+using Elders.Web.Api;
+using System.Web.Http;
+using Discovery.Contracts;
+using System.Collections.Generic;
+using PushNotifications.Contracts.PushySubscriptions;
+using PushNotifications.Contracts;
+using System;
+
+namespace PushNotifications.Api.Controllers.Subscriptions.Commands
+{
+    [RoutePrefix("Subscriptions/PushySubscription")]
+    public class PushyUnSubscriptionController : ApiController
+    {
+        public IPublisher<ICommand> Publisher { get; set; }
+
+        /// <summary>
+        /// UnSubscribes from push notifications with Pushy token
+        /// </summary>
+        /// <param name="model"></param>
+        /// <returns></returns>
+        [HttpPost, Route("UnSubscribe"), Discoverable("PushySubscriptionUnSubscribe", "v1")]
+        public IHttpActionResult UnSubscribeFromPushy(PushySubscribeModel model)
+        {
+            var result = new ResponseResult(Constants.InvalidCommand);
+            if (Urn.IsUrn(model.SubscriberUrn) == false) return this.NotAcceptable($"{nameof(model.SubscriberUrn)} must be URN.");
```

Comment:
why should we check this?
why not have a converter?
shouldn't the urn be urlencoded

Reply:
This was obsolete. It is removed now